coins/monero/tests/runner.rs

```diff
@@ -74,6 +75,18 @@ pub async fn get_miner_tx_output(rpc: &Rpc<HttpRpc>, view: &ViewPair) -> Spendab
   scanner.scan(rpc, &block).await.unwrap().swap_remove(0).ignore_timelock().swap_remove(0)
 }
 
+/// Makes sure the weight and fee match the expected calculation
+pub fn check_weight_and_fee(tx: &Transaction, fee_rate: Fee) {
```

Comment:
This should also be run on TXs sent via monero-wallet-rpc, which we should have in lib.

Reply:
See 4905e6fd3c2eaf6a2264bab2ce00ea2b7e1b8790

If https://github.com/monero-project/monero/pull/8882 is merged then that commit could be reverted

Reply:
Add a TODO in codebase for when 8882 is merged?

Reply:
Add a TODO in codebase for when 8882 is merged?